integration-testing/src/functionalTest/kotlin/kotlinx.atomicfu.gradle.plugin.test/cases/MppProjectTest.kt

```diff
@@ -41,4 +41,22 @@ class MppProjectTest {
         assert(mppSample.cleanAndBuild().isSuccessful)
         mppSample.checkConsumableDependencies()
     }
+
+    @Test
+    fun testMppNativeWithEnabledIrTransformation() {
+        mppSample.enableNativeIrTransformation = true
+        assert(mppSample.cleanAndBuild().isSuccessful)
```

Comment:
The same issue I mentioned in wasm-support-PR: please use assertTrue instead of assert.

Reply:
Fixed